src/main/java/us/ihmc/commons/thread/ThreadTools.java

```diff
@@ -125,6 +127,56 @@ public static void sleepForever()
       }
    }
 
+   /**
+    * Similar to {@link #sleepSeconds(double)}, but uses {@link LockSupport#parkNanos} to sleep.
+    * {@link LockSupport#parkNanos} is more accurate than {@link Thread#sleep}.
+    * The requested sleep is guaranteed to be at least as long as the requested
+    * amount and can be up to a nanosecond longer.
+    * <p>
+    * Also, while {@link #sleepSeconds(double)} swallows interrupts,
+    * this method will return upon being interrupted
+    * and the calling thread's interrupt status will be preserved.
+    */
+   public static void park(double seconds)
```

Comment:
The `MissingThreadTools.sleep` method. 

I considered replacing the logic of the `ThreadTools.sleep` methods, but I think the two methods deserve to be separate since they behave slightly differently: 
- `MissingThreadTools.sleep` returns upon interrupt, whereas `ThreadTools.sleep` swallows it (this is an important difference for some code). 
- `MissingThreadTools.sleep` uses `LockSupport.park`, meaning the parked thread can be un-parked using `LockSupport.unpark` (although we never do that). 

How do we feel about this?

Reply:
Yeah, I like this reasoning